config/cmdline

```diff
@@ -1 +1 @@
-BOOT_IMAGE=/vmlinuz-5.10.0-11-amd64 root=/dev/mapper/vroot ro quiet vt.handoff=7 fsck.mode=skip verity.hashdev=/dev/mapper/Vx--vg-hashes verity.rootdev=/dev/mapper/Vx--vg-root verity.hash=
+root=/dev/mapper/vroot ro quiet vt.handoff=7 fsck.mode=skip verity.hashdev=/dev/mapper/Vx--vg-hashes verity.rootdev=/dev/mapper/Vx--vg-root verity.hash=
```

Comment:
this removal is purposeful? Just double checking.

Reply:
I _think_ it's unnecessary and may have been breaking GRUB. I'm testing non-scanner devices atm to verify that it was unnecessary.

Reply:
Confirmed it's unnecessary